### ISSUE TITLE
Internalization of title and message

### ISSUE THREAD

**Issue by pinturic** [phonegap/phonegap-plugin-push#737]
Is there any way to add internationalisation to the rendered notifications?
I was thinking something like a {key, value} bundle to be passed to the _init_ function and this bundle should be used like a mini template to render the localized title and message.

**Comment by macdonst** [comment]
@pinturic you are better off finding out what the preferred language of your users are and when you send out push messages you send one to all the English users then one to all the French users for instance.

You can look it up using:

`var language = window.navigator.userLanguage || window.navigator.language;`

**Comment by pinturic** [comment]
You are right this could be one of the options. Bu the logic in our system that sends the notification should remain is language-agnostic delegating the rendering to the client (since already contains our language bundle).

**Comment by pinturic** [comment]
I was thinking something like:

var push = PushNotification.init({
      android: {
        senderID: GCM_SENDER_ID,
        forceShow: true:
        **messages**: {
             "RECEIVED": "You have received",
             "NEW_ALERT: "new alert" 
        }
       }
    });

and when a message with a title: "RECEIVED 1 NEW_ALERT" is received it is rendered like "You have received 1 new alert".
